platform/web/src/useAuth.ts

```
@@ -0,0 +1,40 @@
+import { useEffect, useState } from 'react';
+
+export interface AuthUser {
+  id: string;
```

ESLint

[error] Missing JSDoc comment.

(jsdoc/require-jsdoc)